selfdrive/car/hyundai/hyundaican.py

```diff
@@ -78,27 +78,27 @@ def create_lfahda_mfc(packer, enabled, hda_set_speed=0):
   }
   return packer.make_can_msg("LFAHDA_MFC", 0, values)
 
-def create_acc_commands(packer, enabled, accel, jerk, idx, lead_visible, set_speed, stopping, gas_pressed):
+def create_acc_commands(packer, enabled, accel, jerk_upper, jerk_lower, idx, lead_visible, set_speed, stopping, gas_pressed):
   commands = []
 
   scc11_values = {
     "MainMode_ACC": 1,
     "TauGapSet": 4,
     "VSetDis": set_speed if enabled else 0,
     "AliveCounterACC": idx % 0x10,
-    "ObjValid": 1 if lead_visible else 0,
-    "ACC_ObjStatus": 1 if lead_visible else 0,
+    "ObjValid": 1, # close lead makes controls tighter
+    "ACC_ObjStatus": 1, # close lead makes controls tighter
     "ACC_ObjLatPos": 0,
     "ACC_ObjRelSpd": 0,
-    "ACC_ObjDist": 0,
+    "ACC_ObjDist": 1, # close lead makes controls tighter
```

Comment:
I think someone said this causes a lead to show up on their dash now, is this permanent, and do we care?

Reply:
ObjGap below is what causes the lead to be displayed